packages/terra-search-field/src/SearchField.jsx

```diff
@@ -231,6 +238,10 @@ class SearchField extends React.Component {
     } else {
       additionalInputAttributes.defaultValue = defaultValue;
     }
+    /* If block to set autoFocus based on prop value. */
+    if (enableAutoFocus === true) {
```

Comment:
I'd simply do  ``` additionalInputAttributes.autoFocus = enableAutoFocus; ``` and remove the `if` block.

Reply:
I am assuming this condition is added to make sure auto-focus attribute gets added to search-input only when it is required i.e. when enableFocus is true. Because adding attribute autofocus to search-input by default will make input to be auto-focused even for enableFocus = false.

Reply:
> Because adding attribute autofocus to search-input by default will make input to be auto-focused even for enableFocus = false.

No, it does not. The condition is redundant imo.

Reply:
@nramamurth . Removing if works perfectly fine. But I guess we'll have to wait for UX review to find out whether this enhancement is required.